src/include/sof/lib/dai.h

```diff
@@ -417,6 +420,18 @@ static inline int dai_get_fifo(struct dai *dai, int direction,
 	return ret;
 }
 
+/**
+ * \brief Get Digital Audio dma srcid
+ */
+static inline int dai_get_srcid(struct dai *dai, int direction,
+			       int stream_id)
+{
+	int ret = dai->drv->ops.get_srcid(dai, direction, stream_id);
```

Comment:
return 0 here if ops.get_srcid is NULL.

Reply:
Yes. It should be 0 if the platform is not 8ULP. Then edma won't set srcid.